docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

```diff
@@ -174,6 +174,13 @@ The options in `Global` section are used for openstack-cloud-controller-manager
   For example, this option can be useful when having multiple or dual-stack interfaces attached to a node and needing a user-controlled, deterministic way of sorting the addresses.
   Default: ""
 
+### Router
+
+* `router-id`
+  Required. Specifies the Neutron router ID to manage Kubernetes cluster routes.
+* `extraroute-atomic`
```

Comment:
maybe in the future we can check through `neutron ext-list` to detect whether it's enabled or not ?

Reply:
that's what I'm investigating right now.

Reply:
done. my tests show overall routes reconciliation latency improvements from 2-5 minutes down to 10 seconds for a cluster of 16 nodes.

ready for review.